test/e2e/scheduling/resource_quota.go

```diff
@@ -380,14 +384,18 @@ var _ = SIGDescribe("ResourceQuota", func() {
 		usedResources[v1.ResourcePods] = resource.MustParse("1")
 		usedResources[v1.ResourceCPU] = requests[v1.ResourceCPU]
 		usedResources[v1.ResourceMemory] = requests[v1.ResourceMemory]
+		usedResources[v1.ResourceName(v1.SharedResourceRequestsPrefix+extendedResourceName)] = requests[v1.ResourceName(extendedResourceName)]
 		err = waitForResourceQuota(f.ClientSet, f.Namespace.Name, quotaName, usedResources)
 		Expect(err).NotTo(HaveOccurred())
 
 		By("Not allowing a pod to be created that exceeds remaining quota")
 		requests = v1.ResourceList{}
+		limits = v1.ResourceList{}
 		requests[v1.ResourceCPU] = resource.MustParse("600m")
 		requests[v1.ResourceMemory] = resource.MustParse("100Mi")
-		pod = newTestPodForQuota(f, "fail-pod", requests, v1.ResourceList{})
+		requests[v1.ResourceName(extendedResourceName)] = resource.MustParse("2")
+		limits[v1.ResourceName(extendedResourceName)] = resource.MustParse("2")
+		pod = newTestPodForQuota(f, "fail-pod", requests, limits)
```

Comment:
I think this part of test verifies a pod can't be created based on cpu quota. Can we test separately that a pod can't be created with only its extended resource requests exceeding quota (i.e., creating a separate pod that only requests 2 extended resources)?

Reply:
I meant to do so, but was afraid someone might feel it a little redundant to add ER related cases everywhere on the other hand. We could already ensure the functionality by
1. Validate ER usage correctly calculated;
2. Validate it works well for exceeding quota scenario through an example resource(CPU here),
which may seems not intuitive enough though.

Since you think it better to add case specifically for the scene, Let's have one :)